autoload/ctrlsf.vim

```diff
@@ -232,3 +232,16 @@ endf
 func! ctrlsf#ClearSelectedLine() abort
     call ctrlsf#hl#ClearSelectedLine()
 endf
+
+" Toggle()
+"
+func! ctrlsf#Toggle() abort
+    let ctrlsfwinnr = bufwinnr("__CtrlSF__")
+    if ctrlsfwinnr != -1
+        call ctrlsf#preview#ClosePreviewWindow()
+        call ctrlsf#win#CloseMainWindow()
```

Comment:
Maybe `ctrlsf#Quit()` is better. I can understand your efforts of keeping using low-level methods, but I think it's ok using `ctrlsf#Quit()` here, that is exactly what we are doing.